### DISCUSSION TITLE
Nspanel app not fully working

### DISCUSSION THREAD

**Discussion by kroon040** [seaky/nspanel_pro_tools_apk#43] [Q&A]
Hi,

Thnx for v2 release. I want to use the app for the display to set on and of of specific times. I set the times, but the panel stays on. Is there anything else what I should do of can check what goes wrong?

Nspanel pro v1.33

**Comment by seaky** [comment]
Check "Display sleep" under display/screen category if it is "Never sleep" it wont sleep. Change value to 15seconds.

There is a visual bug and the "Display sleep" is not updating. Workaround: switch between tabs and it will be updated

**Comment by seaky** [comment]
and also check OS System -> Developer options > Stay awake. If it is on. It overrides everything.

**Comment by kroon040** [comment]
I had never sleep on, in both options. I don't want the screen to dim so much, with sleep

**Comment by seaky** [comment]
Sorry I dont quite understand what behaviour you want. 

If you set Display sleep option to eg 15 sec it will turn of completely after 15 sec. 
If you want to keep the screen always on within a certain period you can use always-on feature which will wake up the device in the given period.

If you want to just dim the screen and on a given time light-up the screen to prescribed brightness it is not implemented yet it will be available in 2.1

**Comment by kroon040** [comment]
I want the display complete on with brightness 90%  between 07:00-23:00. Between 23:00-07:00 it must be off.

If i set in android never sleep it will stay on and I can do 
adb shell input keyevent 26 for on and off

**Comment by seaky** [comment]
Ohh I see I think I figured out what the problem is...

If you set the time earlier than the current time it will take effect tomorrow. It's a bug or  behaviour :)

**Comment by seaky** [comment]
I'll fix it in 2.0.1 soon
